### PR TITLE
[docs] add athena data table creation docs

### PR DESCRIPTION
**Pull request by jacknagz**
to: @ryandeivert 
cc: @airbnb/streamalert-maintainers 
size: small

## Background

We recently added support for searching data via Athena and Firehose, but there are no steps on how to set this up.

## Changes

* Add some DOCS

## Testing

Locally with the test_the_docs.sh script

### CODE REVIEW

File: docs/source/athena-deploy.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ The name of the Lambda function is Athena Partition Refresh.
 
 The purpose of this function is to periodically refresh Athena tables to enable the search-ability of data.
 
-The default refresh interval is 10 minutes.
+The default refresh interval is 10 minutes, this can be adjusted based on throughput.
```
Comment (mime-frame) [inline docs/source/athena-deploy.rst]: Add any context the user should reason about when trying to decide what # they should pick. Example: Why wouldn't everyone want to just put this to 1 minute in order to have data available the quickest?

File: docs/source/athena-setup.rst [hunk 2]
```diff
@@ -49,13 +49,30 @@ Next, create the ``streamalert`` database:
...
+
+.. code-block:: bash
+
+  $ python manage.py athena create-table \ 
+    --type data \
+    --bucket <prefix>.streamalert.data \
+    --refresh_type add_hive_partition \
+    --table_name <log_name>
+
+Note: The log name above is representative of an enable log source to your StreamAlert deployment.
```
Comment (mime-frame) [inline docs/source/athena-setup.rst]: `an enabled log source in your`

File: docs/source/athena-setup.rst [hunk 3]
```diff
@@ -49,13 +49,30 @@ Next, create the ``streamalert`` database:
...
+
+For example, if you have cloudwatch in your sources, you would want to create tables for all possible subtypes.  This includes ``cloudwatch_events`` and ``cloudwatch_flow_logs``.  Also notice that ``:`` is substituted with ``_``; this is due to Hive limitations on table names.
```
Comment (mime-frame) [inline docs/source/athena-setup.rst]: `if you have Cloudwatch in your sources`

Comment (mime-frame) [reply docs/source/athena-setup.rst]: Include a link to what other characters are not allowed in a table name.

Comment (jacknagz) [reply docs/source/athena-setup.rst]: you actually would not stylize `CloudWatch`, because that's not a valid schema name (it's case sensitive)

Comment (ryandeivert) [reply docs/source/athena-setup.rst]: maybe quote it if it's representing a raw string?

Comment (jacknagz) [reply docs/source/athena-setup.rst]: @ryandeivert that works for me

**Comment by mime-frame** [comment]
Pull Request number......

image

**Comment by mime-frame** [comment]
LGTM after suggested changes are made and you update your branch

**Pull request merged**